book/website/community-handbook/enter-the-project.md

```diff
@@ -0,0 +1,89 @@
+(ch-enter-the-project)=
+
+# How to enter *The Turing Way*
+
+*The Turing Way* is an open source project that involves a diverse community of researchers in making data science reproducible, ethical and inclusive for everyone.
+*The Turing Way* book is hosted on GitHub under CC-BY 4.0 license. 
+It is open for anyone to use, contribute to and share. 
+
+The Online Book is the visible product of the work we do in the project, but, the process of collaborative development with the help of its community is what we value the most.
+The backbone of this process lies in supporting its community by providing an open, accessible and inclusive space where they are appreciated, upskilled and given opportunities to contribute to the project in a way that is most meaningful for them.
+In turn, these members support each other in the community, collaborate on mutually beneficial ideas and resources, co-develop chapters, review and improve existing materials, share them with the wider community, make new connections and develop a sense of belonging and shared-ownership in the project.
+
+Since 2019, the project has grown into an international community of people with diverse backgrounds, research domains, expertise, knowledge and lived-experiences. 
+Simultaneously, multiple pathways have evolved that interested members can take to engage in the project.
+Though most of these pathways have been mentioned in various chapters, we have also noticed that often it is hard for people to enter an ongoing project when they don't know what their role in the community would look like.
+
+(ch-overview)=
+## Overview of Resources You Can Use
+
+First, we provide an overview of different platforms, community-led efforts, possible roles and opportunities for collaboration in the community.
+
+In the next subchapters, you will find descriptions of different resources mentioned in the overview table.
+Then we describe other ongoing efforts in the project that you can use to join and ideas for the kind of initiatives you can lead while collaborating with others in continuing your journey within *The Turing Way*.
+Finally, we share some personal motivations of our current members for participating in this project and aspects of their work they find meaningful and beneficial.
+
+## 1. Get to Know the Project
```

Comment:
Should this be a level three subheading? (i.e. ###). It looks like this is a part of the `overview of resources you can use` section.